### ISSUE TITLE
enhancement

### ISSUE THREAD

**Issue by AMARAVATHIVSRKPRASAD** [microsoft/azure-boards-estimate#313]
We are facing some challenges while using Estimate option for poker planning.  Could you please help with below queries?

	Is it possible to archive the session for future reference once the poker planning is done.
	Can we limit the authorization to reveal the card to a specific person( who created the session) instead of everyone?
	Can we make mandatory to choose card whoever is in session?
	Poker planning has time limitation w.r.t to session. If we take more time for then we need to re-do the planning. Could you please help with existing timeout window detail and any possibility to increase the timeout for same?
	During card selection, Is it possible for you to add two new card along with existing card set?

Requirement  to add 3 more cards in below highlighted card set.
•	Card 1 with value 11
•	Card 2 with value 12
•	Card 3 with symbol ?
![image](https://github.com/microsoft/azure-boards-estimate/assets/112699369/344a7a5e-ebd6-4147-b09e-1114c0d22395)

**Comment by BartoszDaniszewski** [comment]
I support the last point of this enhancemnt.
I added separate issue for it:
https://github.com/microsoft/azure-boards-estimate/issues/329
